src/Tasks/ResolveManifestFiles.cs

```diff
@@ -380,6 +382,29 @@ private void GetOutputAssemblies(List<PublishInfo> publishInfos, List<ITaskItem>
                 {
                     if (!IsFiltered(item))
                     {
+                        // ClickOnce for .NET 4.X should not publish duplicate satellite assemblies.
+                        // This will cause ClickOnce install to fail. This can happen if some package
+                        // decides to publish the en-us resource assemblies for other locales also.
+                        if (!LauncherBasedDeployment && satelliteRefAssemblyMap.ContainsItem(item))
+                        {
+                            Debug.Assert(false, $"Duplicate satellite assembly '{item.ItemSpec}' skipped in _managedAssemblies");
```

Comment:
I'm not sure I understand these asserts. Is it a bug in the task if we get to this point? If it's a bug in the user's projects or an unusual condition, I don't think the assert is appropriate.

Reply:
It's about unusual satellite assemblies in file/nuget references. I have removed the asserts as it's not a bug in the task.